pkg/build/builder/util_linux.go

```diff
@@ -98,3 +104,24 @@ func getCgroupParent() (string, error) {
 	glog.V(6).Infof("found cgroup values map: %v", cgMap)
 	return extractParentFromCgroupMap(cgMap)
 }
+
+// getSELinuxLabel retrieves the selinux label associated w/ this container.
+func getSELinuxLabel() (string, error) {
+	f, err := os.Open("/proc/mounts")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	s := bufio.NewScanner(f)
+
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+		if match := selinuxLabelPattern.FindStringSubmatch(s.Text()); match != nil {
+			glog.V(6).Infof("found selinux labels: %v", match[1])
+			return fmt.Sprintf("label=level:%s", match[1]), nil
+		}
+	}
+	return "", nil
```

Comment:
Do we want to return an error if we don't find a match or just skip adding SecurityOpt if we don't find it adding a warning in the log?

Reply:
yeah i'll add a warning and skip.